src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -32,7 +32,7 @@ const TRACK_HEIGHT = 24;
 
 export type ProteinDomainImageProps = {
   transcriptId: string;
-  refCDSLength: number;
+  longestProteinLength: number;
```

Comment:
Same as comment regarding names in the file that's currently called `CollapsedExonsImage.tsx`

Reply:
✔️